packages/dds/tree/api-report/tree.alpha.api.md

```diff
@@ -596,6 +610,11 @@ export interface NodeSchemaOptions<out TCustomMetadata = unknown> {
     readonly metadata?: NodeSchemaMetadata<TCustomMetadata> | undefined;
 }
 
+// @alpha @sealed
+export interface NodeSchemaOptionsAlpha<out TCustomMetadata = unknown> extends NodeSchemaOptions<TCustomMetadata> {
```

Comment:
It seems like users are suppose to create an instance of this to pass into your APIs? If so, marking it as sealed means they are not allowed to do that, and instead have to get the instance of it from one of our APIs. THat seems wrong.

Its use case seems more like the opposite, input, with gets the opposite compatability testing (variance wise).